Mlem/App/Configuration/User Settings/Settings.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Settings.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-08-07.
+//  Adapted from https://fatbobman.com/en/posts/appstorage/
+//
+
+import Foundation
+import MlemMiddleware
+import SwiftUI
+
+// This has to be ObservableObject because Observed currently does not allow @AppStorage properties without @ObservationIgnored
```

Comment:
```suggestion
// This has to be ObservableObject because @Observable currently does not allow @AppStorage properties without @ObservationIgnored
```